pkg/solver/che_routing.go

```diff
@@ -48,6 +48,13 @@ const (
 
 var (
 	configMapDiffOpts = cmpopts.IgnoreFields(corev1.ConfigMap{}, "TypeMeta", "ObjectMeta")
+
+	defaultIngressAnnotations = map[string]string{
```

Comment:
probably I just miss something but should it be declared twice? Here and in pkg/gateway/ingress.go

Reply:
Huh, good catch. I was basing the difference on the property descriptions in the `che.properties` - https://github.com/eclipse/che/blob/69fa3bf82e7aaeb142ebbdc6986f2fc96618dfa0/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L457 (which governs workspace deployments) vs the gateway deployment, e.g. https://github.com/eclipse-che/che-operator/blob/main/pkg/deploy/ingress.go#L93.

But it turns out `che-operator` uses `true` for the `ingress.kubernetes.io/ssl-redirect` in both cases - e.g. https://github.com/eclipse-che/che-operator/blob/b7148052891215a809d362d3d27aaed56ffcc73b/pkg/deploy/server/che_configmap.go#L282.

I will change this to use the single set of annotations.